corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -64,14 +75,12 @@
 
   <script type="text/html" id="js-fd-form-actions">
     <div class="btn-group pull-right">
-      <a class="btn btn-form-action track-usage-link" href="{% url "app_form_summary" domain app.id %}"
-         data-category="App Builder" data-action="Form Action Link" data-label="App Summary"
-      >
+      <a class="btn btn-form-action track-usage-link custom-button-style" href="{% url "app_form_summary" domain app.id %}"
```

Comment:
in bootstrap 5 we could achieve this with style utils, so please leave a comment here that we should replace the <css class> with bootstrap 5 utils after migration.
suggested name: `dark-bg-btn`

Reply:
Add comment in 6a3f58a4f0590a69226350824423c44513314f0c